scripts/quests/windurst/Making_Amends.lua

```diff
@@ -0,0 +1,94 @@
+-----------------------------------
+-- Making Amends
+-----------------------------------
+-- !addquest 2 3
+-- Hakkuru-Rinkuru : !pos -111 -4 101 240
+-----------------------------------
+require('scripts/globals/interaction/quest')
+require('scripts/globals/npc_util')
+require('scripts/globals/quests')
+require('scripts/globals/titles')
+require('scripts/globals/zone')
+-----------------------------------
+
+local quest = Quest:new(xi.quest.log_id.WINDURST, xi.quest.id.windurst.MAKING_AMENDS)
+
+quest.reward =
+{
+    fame     = 150,
+    fameArea = xi.quest.fame_area.WINDURST,
+    title    = xi.title.QUICK_FIXER,
+    gil      = 1500,
```

Comment:
Is the correct gil for the time frame?

Reply:
Yeah, it seems like a lot but it's not repeatable. None of these quests ever had rewards changed, I double checked.